package.json

```diff
@@ -49,7 +49,7 @@
         "build:prod:dist": "NODE_ENV=production webpack --config scripts/webpack.config.js --mode production",
         "build:prod:examples": "LANGUAGE=en-US REACT=true NODE_ENV=production node --max_old_space_size=8192 node_modules/react-styleguidist/bin/styleguidist.js build --config scripts/styleguide.config.js --mode production",
         "build:prod:npm": "BABEL_ENV=production OUTPUT=dist LANGUAGE=en-US REACT=true yarn build:prod:dist",
-        "build:variables": "node ./scripts/build-style-vars.js ./src/styles/_variables.scss ./src/styles/variables.js ./src/styles/variables.json",
```

Comment:
I think we'd still want people to import `variables.json` rather than each individual component of it.

Reply:
So far it's only been used in the `styleguide.config.js` file due to ES module import issues. The issue is that the build-style-vars script can't parse transient variables, so a root-level scss file that reexports other files gets converted to an empty JS/JSON file. I worked around it for the ES module by converting to a manual reexport of the submodules.

Reply:
🤔 

```
cat `find src/styles/constants -name '*.scss'` > src/styles/variables.scss | node ./scripts/build-style-vars.js src/styles/variables.scss && rm src/styles/variables.scss
```